aws-cloudformation/aws-cloudformation-resource-providers-greengrassv2
# Initial commit for AWS::GreengrassV2::ComponentVersion

*Description of changes:*
1. The initial commit of the GreengrassV2 ComponentVersion resource type. Internally, it creates/read/delete a DDB table. Since the DDB table creation is also an async operation, we use it to simulate the component version creation.
2. Add `resource-role.yaml ` into the .gitignore file.

*Testing*
1. Make sure the mvn build & test successfully.
2. Deploy this resource type to the personal stack and verify the DDB table is created.
3. Delete the stack and verify the DDB table is removed.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
